### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by rotsen18**

### CODE REVIEW

**Review by KatiukhaO — APPROVED** [review]

File: db/models.py [hunk 1]
```diff
@@ -1 +1,10 @@
 from django.db import models
+
+
+class Genre(models.Model):
+    name = models.CharField(max_length=255)
```
Comment (Magus6784) [inline db/models.py:5]: Char field name, name of the genre with the maximum length of 255 characters, and name must be unique

**Review by Magus6784 — CHANGES_REQUESTED** [review]

File: db/models.py (hunk 1 above)
Comment (Magus6784) [reply db/models.py:5]: This is literally words from task

Comment (Magus6784) [reply db/models.py:5]: Lol, where are they?!

**Review by Magus6784 — APPROVED** [review]
Change's on your side

**Pull request closed without merging**